script/upgrade/Mode-100-to-100.s.sol

```diff
@@ -0,0 +1,351 @@
+import {Script} from "forge-std/Script.sol";
+import {Test, console2 as console} from "forge-std/Test.sol";
+
+import {Multisig} from "@aragon/multisig/Multisig.sol";
+import {IDAO} from "@aragon/osx/core/dao/IDAO.sol";
+
+import {GaugesDaoFactory, GaugePluginSet, DeploymentParameters, Deployment, TokenParameters, DAO, IGaugeVote} from "@factory/GaugesDaoFactory.sol";
+import {VotingEscrow, Lock, QuadraticIncreasingEscrow, ExitQueue, SimpleGaugeVoter, SimpleGaugeVoterSetup, ISimpleGaugeVoterSetupParams} from "@voting/SimpleGaugeVoterSetup.sol";
+import {SimpleGaugeVoterV1_1_0} from "@voting/SimpleGaugeVoter_v1_1_0.sol";
+
+import {Upgrades} from "@foundry-upgrades/LegacyUpgrades.sol";
+import {Options} from "@foundry-upgrades/Options.sol";
+
+contract UpgradeModeTo110 is Script, Test {
+    /////////////////////////////////////////////
+
+    /////////////////////////////////////////////
+
+    /// @dev metadata for the proposal, pinned to pinata
+    bytes ipfsURI = bytes("ipfs://bafkreicqy5hgf6izqha6hoa6cudup4or5clfnilegyavwuhbr34xlihsea");
+
+    function setAragonSigners() internal {
+        aragonSigners.push(address(0x946138B088524414EEDaf0699BA10d7Fb5673A34));
```

Comment:
What are your thoughts on moving this to a file similar to the Mode signers?

Reply:
Kind of annoying not to have in version control tbh.